articles/machine-learning/how-to-managed-network.md

```diff
@@ -651,8 +651,7 @@ To enable the [serverless spark jobs](how-to-submit-spark-jobs.md) for the manag
     ```
 
     # [Azure portal](#tab/portal)
-
-    Create a new compute instance or compute cluster, which also creates the managed virtual network.
```

Comment:
why removing this line? this statement is accurate

Reply:
@joshharrin I think they want to say "Create a new compute instance or compute cluster, which also creates the managed virtual network. **You need to do xyz to use serverless spark.**